server/index.js

```diff
@@ -8,9 +8,9 @@ const port = 3000;
 const db = require('./db/index.js');
 
 app.get('/:restaurant_id/reviews', (req, res) => {
-  const sqlreq = 'SELECT * FROM reviews WHERE restaurant_id=?';
+  const sqlreq = 'SELECT reviews.*, restaurants.* FROM reviews, restaurants WHERE reviews.restaurant_id=restaurants.restaurant_id AND reviews.restaurant_id=?';
```

Comment:
no need for change, but here is another way you could have written it:
`‘SELECT reviews.*, restaurants.*  
	FROM reviews rev INNER JOIN restaurants res
		ON rev.restaurant_id = res.id
	WHERE rev.restaurant_id=?’;`